### PR TITLE
Api submit genbank

### PR DESCRIPTION
**Pull request by kdp-cloud**
This feature feeds information from a genbank file to the api which stores it in a local database.

### CODE REVIEW

File: server/pyproject.toml [hunk 1]
```diff
@@ -6,12 +6,14 @@ authors = ["James Collier <james.collier@vib.be>"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-fastapi = "^0.70.0"
```
Comment (MaybeJustJames) [inline server/pyproject.toml:9]: Probably shouldn't be a downgrade

**Review by MaybeJustJames — CHANGES_REQUESTED** [review]
Some minor issues. A couple major issues:
1. Old `Diagram GGW.svg`
2. The dependency on `fastapi` was downgraded. Was this intentional?
3. This PR contains 2 database revisions. Can you merge them into 1 please. Be very careful with the revision ids. Also make sure the descriptions are useful (not e.g. `short summary of changes`)

**Review by kdp-cloud — COMMENTED** [review]
Approved all changes.

File: server/pyproject.toml (hunk 1 above)
Comment (kdp-cloud) [reply server/pyproject.toml:9]: ### Issue n°2
As far as I know, there was no reason to downgrade the version of fastapi.
### Issue n°3
The two database revisions were successfully merged without any compiler errors.


Resolved these issues in latest commit (e0f8cc2).

**Review by MaybeJustJames — APPROVED** [review]

**Pull request merged**